sequence_transform.c

```diff
@@ -479,7 +531,9 @@ bool process_sequence_transform(uint16_t keycode, keyrecord_t *record, uint16_t
         // tell QMK to not process this key
         return false;
     } else {
-        // TODO: search for rules
+#ifdef SEQUENCE_TRANSFORM_MISSED_RULES
+        st_find_missed_rule();
+#endif
```

Comment:
it really should check token keys too, like if you used unoptmal sequence

Reply:
You mean always call this, even if we just did a completion?
Can you give an example where this would be needed?

Reply:
`sti@ -> still` and `stil@ -> still` as a regular repeat rule, it should show that you could ve used better(*shorter*) sequence

Reply:
right now this isn't supported. If you type `stil@`, that's what's in the buffer, so it wouldn't reverse match `still` even if we did a rule search after the `@`.